ngraph/frontend/onnx/frontend/include/onnx_frontend/input_model.hpp

```diff
@@ -0,0 +1,33 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <frontend_manager/input_model.hpp>
+#include <onnx_editor/editor.hpp>
+
+namespace ngraph
+{
+    namespace frontend
+    {
+        class FRONTEND_API InputModelONNX : public InputModel
```

Comment:
Consider moving to src subdirectory as an internal detail. Please read another my relevant comment about Places.